lessons/introduction-to-smartpy/14/14.mdx

```diff
@@ -208,4 +208,4 @@ You can also tinker around with the final code on [SmartPy’s online IDE](https
 Read more in depth about: 
 
 1. Technical pov of using `sp.if` over `if` in your smart contract code - under reference manual's "Control & Syntactical Sugar" section [here.](https://smartpy.io/demo/reference.html#_control_and_syntactic_sugar)
-2. Why you cannot use `and`, `or`, `not` in your smart contract code and instead need to use `&`, `|`, `~` respectively - under reference manual's "Boolean" section [here.](https://smartpy.io/demo/reference.html#_booleans)
\ No newline at end of file
+2. Why you cannot use `and`, `or`, `not` in your smart contract code and instead need to use `&`, `|`, `~` respectively - under reference manual's "Boolean" section [here.](https://smartpy.io/demo/reference.html#_booleans)
```

Comment:
@STFinancial I am not able to detect any visible changes here through 2-3 reading. Can you help me in pointing to what was the change here?

Reply:
I don't know why this is being listed as a diff, I also did not make a change here